examples/tests/sdk_tests/std/shared-state.w

```diff
@@ -0,0 +1,33 @@
+bring cloud;
+
+class QueueSim {
+  rawMessages: std.SharedState;
```

Comment:
Something feels wrong about this.

At the minimum, this should be `sim.SharedState` and not `std.SharedState`.

And then maybe it should just be `sim.Object`?

Reply:
Maybe this needs a sync discussion...

Reply:
> Something feels wrong about this.

Just curious, do you mean just with the name, or with other stuff? (FWIW I'm not 100% sold it's the most elegant thing, but also intuitively feels to me like an abstraction we will need in one shape or another). I'm cool with the rename suggestions 👍 

I'll comment that the fact that you have to create `sim.Object` in preflight init feels a bit restrictive to me -- `sim.Object` might be a good candidate for a future phase-independent class. That is, `sim.Object<T>` can hold any `T` that is also phase-independent (like collections, structs, and so on), and it can be declared in preflight or just in inflight if needed. But until we iron out phase-independence, I think this might be adequate.

Reply:
In inflight, an object is just an object, no?